nixos/modules/system/boot/modprobe.nix

```
@@ -34,6 +34,23 @@ with lib;
       type = types.lines;
     };
 
+    boot.initrd.extraModprobeConfig = mkOption {
+      default = "";
+      example =
+        ''
+          options zfs zfs_arc_max=1073741824
+        '';
+      description = ''
+        Does exactly the same thing as
+        <option>boot.initrd.extraModprobeConfig</option>, except
```

I may be wrong, but:

```suggestion
        <option>boot.extraModprobeConfig</option>, except
```

---

You're absolutely right, thanks for catching that.